src/dataviews/helpers/histogram-helper.js

```diff
@@ -2,31 +2,66 @@ var moment = require('moment');
 var _ = require('underscore');
 
 var MOMENT_AGGREGATIONS = {
-  day: 'd',
-  hour: 'h',
+  second: 's',
   minute: 'm',
+  hour: 'h',
+  day: 'd',
+  week: 'w',
   month: 'M',
   quarter: 'Q',
-  second: 's',
-  week: 'w',
   year: 'y'
 };
 
+function isLessThanDays (aggregation) {
+  return aggregation === 'second' || aggregation === 'minute' || aggregation === 'hour';
```

Comment:
+1 being picky I'd move this to an utils with `isShorterThan('days', aggregation)`, and the method would check against the MOMENT_AGGREGATIONS index below the passed time BOOM